Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/ConceptManager.java

```diff
@@ -572,4 +582,11 @@ private int getConceptsCountForConceptList(String listName) {
         alternativeIdService.addAlternativeIds(conceptEntries);
         return conceptEntries;
     }
+
+    @Override
+    public void addWordnetConceptEntry(ConceptEntry entry, String userName)
```

Comment:
now I'm really confused. This method actually doesn't add a concept entry. It's only adding it to the index. If anything there should be a method to add a concept to the index. The list should be set before this is done.